cyber/record/record_viewer.cc

```diff
@@ -58,7 +58,7 @@ bool RecordViewer::IsValid() const {
 
 bool RecordViewer::Update(RecordMessage* message) {
   bool find = false;
-  while (!find) {
+  do {
```

Comment:
Personally I try to avoid `do...while`. Here we can use
`while(!find && (!msg_buffer_.empty() || FillBuffer()))`

Reply:
I think do...while is more readable and maintainable, thanks for your extremely careful review and comments!